### PR TITLE
Add `count` & `sorted`, `itemgetter_default`, use `itemgetter`

### PR DESCRIPTION
**Pull request by 0xnurl**

### CODE REVIEW

File: gamla/functional.py [hunk 1]
```diff
@@ -27,6 +27,11 @@ def identity(x):
 do_breakpoint = curried.do(lambda x: builtins.breakpoint())
 
 
+count = toolz.count
+
+sorted = currying.curry(sorted)
```
[the remark this answers is not in the document]
Comment (0xnurl) [reply gamla/functional.py:32]: np, but i think it was requested in the migration spreadsheet

[the remark this answers is not in the document]
Comment (nitzo) [reply gamla/functional.py:32]: reversed ? or curry ?
curry was requested as a nested function iiuc

[the remark this answers is not in the document]
Comment (nitzo) [reply gamla/functional.py:32]: maybe we can have two versions:
sorted and sorted_reversed

File: gamla/functional.py [hunk 2]
```diff
@@ -27,6 +27,23 @@ def identity(x):
 do_breakpoint = curried.do(lambda x: builtins.breakpoint())
 
 
+count = toolz.count
+
+
+def sort(key: Callable):
```
Comment (uriva) [inline gamla/functional.py:33]: make a `sort_by` and use it to make `sort`

File: gamla/functional.py [hunk 3]
```diff
@@ -27,6 +27,23 @@ def identity(x):
...
+    def inner(seq: Iterable):
```
Comment (uriva) [inline gamla/functional.py:34]: call this by the same name, to allow for easier debugging

File: gamla/functional.py [hunk 4]
```diff
@@ -27,6 +27,23 @@ def identity(x):
...
+        return sorted(seq, key=key)
+
+    return inner
+
+
+def sort_reversed(key: Callable):
+    def inner(seq: Iterable):
```
Comment (uriva) [inline gamla/functional.py:41]: call this by the same name

File: gamla/functional_generic.py [hunk 5]
```diff
@@ -436,6 +436,14 @@ async def excepts(*args, **kwargs):
 )
 
 
+def itemgetter_default(default, attr):
```
Comment (uriva) [inline gamla/functional_generic.py:439]: consider `itemgetter_with_default` to match `get_in_with_default`

Comment (uriva) [inline gamla/functional_generic.py:439]: move to `functional.py`, it doesn't need `async` support. Use `toolz.excepts`

**Review by uriva — APPROVED** [review]

**Review by uriva — APPROVED** [review]

**Pull request merged**